package.json

```diff
@@ -24,12 +24,13 @@
     "test": "yarn jest",
     "test:watch": "yarn jest --watchAll",
     "storybook": "start-storybook -p 6006",
-    "storybook-ssl": "start-storybook -p 6006 --https --ssl-cert ./server.cert --ssl-key ./server.key"
+    "storybook-ssl": "start-storybook -p 6006 --https --ssl-cert ./server.cert --ssl-key ./server.key -s ./public"
   },
   "repository": "cognitedata/gearbox.js",
   "homepage": "https://github.com/cognitedata/gearbox.js#readme",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cognite/3d-viewer": "^4.1.1",
```

Comment:
This is maybe a big dependency. Any way they can optionally choose to have it or not?

Reply:
we can add it to optionalDependencies and handle it while using this component. Also we need to add additional info to documentation to inform user about required dependency

Reply:
Yes. I wonder how typescript will handle the `*.d.ts` files if it is an optional dependency